RusimbiPatrick/Prolite
# #167196332 Ch setup dependencies and tests

### What does this PR do?
This PR sets up requires dependencies and a testing environment.
### Description of Task to be completed?

1. The testing environment should be configured
2. **The following dependencies should be preset:**
"dependencies": {
    "@babel/polyfill": "^7.4.4",
    "@babel/register": "^7.4.4",
    "@hapi/joi": "^15.1.0",
    "bcryptjs": "^2.4.3",
    "body-parser": "^1.19.0",
    "cloudinary": "^1.14.0",
    "connect-multiparty": "^2.2.0",
    "cookie-parser": "^1.4.4",
    "core-js": "^3.1.4",
    "dotenv": "^8.0.0",
    "esm": "^3.2.25",
    "express": "^4.17.1",
    "jsonwebtoken": "^8.5.1",
    "moment": "^2.24.0"
  },
  "devDependencies": {
    "@babel/cli": "^7.5.0",
    "@babel/core": "^7.5.4",
    "@babel/preset-env": "^7.5.4",
    "chai": "^4.2.0",
    "chai-http": "^4.3.0",
    "coveralls": "^3.0.4",
    "cross-env": "^5.2.0",
    "eslint": "^6.0.1",
    "eslint-config-airbnb-base": "^13.2.0",
    "eslint-plugin-import": "^2.18.0",
    "mocha": "^6.1.4",
    "mocha-lcov-reporter": "^1.3.0",
    "nodemon": "^1.19.1",
    "nyc": "^14.1.1"
  }
}

### How should this be manually tested?
**By using any text editor**,  Check if the dependencies mentioned above are present in **package.json** file.
**By using terminal**,  Check if the testing environment  is well configured i.e, _If there are tests passing or failing_
